clients/client-sdk/src/models/apiVersion.ts

```diff
@@ -0,0 +1,4 @@
+export enum ApiVersion {
```

Comment:
please also apply this in the specific routes instead of using a string

Reply:
the property from ClientConfig is used in the url strings